azuki-core/src/main/kotlin/com/anaplan/engineering/azuki/core/runner/JUnitScenarioRunner.kt

```diff
@@ -367,7 +374,7 @@ class JUnitScenarioRunner<
     }
 
     private fun parameterize(baseRuns: List<ScenarioRun<AF, CF, QF, AGF>>): List<ScenarioRun<AF, CF, QF, AGF>> {
-        val parameterPermutations =
+        @Suppress("UNCHECKED_CAST") val parameterPermutations =
```

Comment:
:pick: I hadn't put this in because it's nonsense -- it _is_ checked!

Reply:
Meh, I'm not sure why the tests weren't running without suppressing it. 🤷